src/contact.ts

```diff
@@ -55,9 +64,8 @@ export type ContactQueryFilter = {
 
 /**
  * Class Contact
- * blabla...
- * **IMPORTANT**
  *
+ * `Contact` is `Sayable`
```

Comment:
You are right, maybe I'm too familiar with Wechaty, so I add examples for all API functions.